datastore/simple/transformers/payload.go

```diff
@@ -0,0 +1,81 @@
+package transformers
+
+import (
+	"time"
+
+	"github.com/teslamotors/fleet-telemetry/protos"
+)
+
+func PayloadToMap(payload *protos.Payload, includeTypes bool) map[string]interface{} {
+	convertedPayload := make(map[string]interface{}, len(payload.Data)+2)
+	convertedPayload["Vin"] = payload.Vin
```

Comment:
we already log vin, do we need to log it here?

Reply:
It's in the payload, so going to include it.